llvm/include/llvm/Passes/TargetPassRegistry.inc

```diff
@@ -92,7 +92,7 @@ if (PIC) {
     return true;                                                               \
   }
 
-PB.registerPipelineParsingCallback([=](StringRef Name, ModulePassManager &PM,
+PB.registerPipelineParsingCallback([&](StringRef Name, ModulePassManager &PM,
```

Comment:
I might be missing something: why was this changed from captured by value to captured by reference.